charts/enterprise-operator/crds/webhook-cluster-role.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
This is a newer way to install this; is there any notes/docs we can add?
How does this effect watch namespace?

Reply:
This is not a new way. These are the exact same resources we have in Enterprise in the [public repository](https://github.com/mongodb/mongodb-enterprise-kubernetes/blob/master/helm_chart/crds/webhook-cluster-role.yaml).

Now, I don't think this belongs in `crds/` directory, but will leave it there for now. We can revisit this later.